README.md

```diff
@@ -107,6 +106,103 @@ print(myShoppingList)
+
+```
+0 => "Bread",
+1 => "Butter",
+2 => "Cheese",
+3 => "Lettuce",
+4 => "Tomatoes",
+```
+
+This is a collection of five string values. The item on the right hand side of the `=>` symbol is a string value, and the number to the left it's position in the array. Note that an array starts from 0, and not 1\. This is common in programming languages, so for a five item array, the last item will actually be at position number 4.
+
+Whilst arrays share similar concepts and principles across programming languages, you create and use them differently in different languages.
+
+## Creating an Array
+
+Here's how to create an array of values in Swift:
+
+```swift
+var list = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+In this case, you are creating a variable with the `var` keyword called 'list'. Swift will decide on the type of values allowed in the array for you as you add them.
+
+To be more precise, you can declare the type of values allowed in the array explicitly:
+
+```swift
+var shoppingList: [String] = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Here you are creating a variable with the `var` keyword called 'shoppingList', setting the type of values allowed explicitly to be of `String` and then adding five string values to the array.
+
+Or alternatively (but less used):
+
+```swift
+var shoppingList: Array<String> = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Often an array is initialized first and populated later:
+
+```swift
+var shoppingList: [String]
+shoppingList = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Here you create an array variable called 'shoppingList' that will accept values of type `String`. You then assign five string values to the variable.
+
+## Constant Arrays
+
+As you have used the `var` keyword in all the examples so far, these collections of values are _mutable_. This means you can add, remove or change items in the array. As with other variable types in Swift, to make an array constant, unchangeable or _immutable_, use the `let` keyword.
+
+For example, a shopping list may change, but the ingredients for a cheese sandwich never will:
+
+```swift
+let cheeseSandwich: [String] = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+## Other Values in an Array
+
+As you would expect, arrays can contain values apart from strings, and created the same way. For example, to create an array recording attendance at your Saturday afternoon sandwich meet:
+
+```swift
+var afternoonAttendance: [Int] = [10, 5, 15, 2, 12]
+```
+
+## Accessing Items in an Array
+
+You have an array of values, to access an individual item, you need to know it's location in the collection. For example, in the `shoppingList` array, you want to print the value of the item in the second position, starting from 0, this is position 1\. Below, you assign the value in that position to a variable, and print the value of the variable.
+
```

Comment:
Continuing with the narrative, have a WHY?

Just like the items in our shopping list, elements in an array are associated with a number that represents their order. In programming, this number is called an index. While humans typically start their lists at "1.", arrays begin their indexes at 0 (zero). So, the first item in array will always be "at index 0". 

For example, we have our shopping list and we need now to access items from that list. For instance... someone asks "Whats the first item on the shopping list we need?". How can we access this item? We know how to create an array, we can see the various elements an array, but we don't know how to access them individually". You can think of an array as numbered list as if these ingredients were written down on a piece of paper as follows:
1. Bread
2. Butter
3. Cheese
4. Lettuce
5. Tomatoes

But!! Although an array is considered an ordered list... it doesn't begin with the number 1, it begins with the number 0 as arrays are zero-based. So imagine now that the list looks like this:

ZERO. Bread
ONE. Butter
TWO. Cheese
THREE. Lettuce
FOUR. Tomatoes

So to access the first item of our array, we do so using the following syntax:
```swift
let firstItem = shoppingList[0]  // Bread
```